doc/api/process.md

```diff
@@ -1642,6 +1642,40 @@ This function is only available on POSIX platforms (i.e. not Windows or
 Android).
 This feature is not available in [`Worker`][] threads.
 
+## process.nice(inc)
```

Comment:
It seems this section should go between `process.nextTick()` and `process.noDeprecation` ABC-wise.

Reply:
Also, if `inc` can be `undefined` (i.e. omitted), then the signature need to be:

`## process.nice([inc])`.

Reply:
Done. Didnt know about the sorting, thanks! bd1e0860a4c16377f90eada353d786f004bc1224